examples/cli/compression/README.md

```diff
@@ -3,19 +3,38 @@
 Website gzip compression makes it possible to reduce the file size of a file
 to roughly 30% of its original size before the files are sent to the browser.
 
+It is enabled by default.
+
+## `--compress`
+
 To run this example, run this command in your console or terminal:
 
 ```console
 npm run webpack-dev-server -- --open --compress
 ```
 
-## What should happen
+### What should happen
 
 1. The script should open `http://localhost:8080/`.
 2. Files being sent to the browser from the `webpack` bundle should be gzipped.
 3. Open the console in your browser's devtools and select the _Network_ tab.
 4. Find `bundle.js`. The response headers should contain `Content-Encoding: gzip`.
 
+## `--no-compress`
+
+To run this example, run this command in your console or terminal:
+
+```console
+npm run webpack-dev-server -- --open --no-compress
```

Comment:
we run webpack serve right?

Reply:
Actually it is script https://github.com/webpack/webpack-dev-server/blob/88468b67ed3f66470acc7816a25e49e07a0fadbf/package.json#L32

Reply:
looks good then

Reply:
Let's change it to `npx webpack serve` in future PRs

Reply:
will take care